.github/workflows/bundle.yml

```diff
@@ -0,0 +1,155 @@
+name: Bundle
+
+on:
+  push: # All branches, but appropriate paths only.
+    # branches:
+    #   - squeak-trunk
+      # - squeak-5.3
+      # - squeak-5.2
+    paths-ignore:
+      - '**.md' # Skip changes in documentation artifacts
+  pull_request:
+    branches:
+      - squeak-trunk
+      # - squeak-5.3
+      # - squeak-5.2
+    paths-ignore:
+      - '**.md'
```

Comment:
If we build on pushes to every branch, why do we need double builds for every PR then?

Reply:
Both push and pull_request should have the same filter. Please read me comments above. This is due to being able to try out the builds in my fork. The one who merges this PR still has to do changes to afterwards.

Reply:
I was rather questioning the existence of either `push` or `pull_request` in general. :-) I am usually going with just `push` to any branch. Or wouldn't this work for forks then?